README.md

```diff
@@ -30,6 +30,18 @@ python ./evaluate.py -m mace_mobilenetv1_nchw -t float32 -k android --target="op
 ```
 Refer to the below instructions for running the `scripts/evaluate.py` script for more information
 
+Additionaly, the script `evaluate_dyn_models.py` could be used to tune and evaluate layers from dynamic models (ssd-resnet, yolov3, faster-rcnn). Below, you can see examples of ssd-resnet evaluation with SD 8 Gen 1 statistics:
```

Comment:
It can probably be in a different section:
```suggestion
## Run models with dynamic shape through VM

Script `evaluate.py` was extended by three models from ONNX model zoo with dynamic shape:[onnx_ssd_resnet34](https://github.com/onnx/models/tree/main/vision/object_detection_segmentation/ssd), [onnx_yolo_v3](https://github.com/onnx/models/tree/main/vision/object_detection_segmentation/yolov3) and [onnx_faster_rcnn](https://github.com/onnx/models/tree/main/vision/object_detection_segmentation/faster-rcnn). They can be inferred by using the following names of the models: `onnx_ssd_resnet34`, `onnx_yolo_v3` and `onnx_faster_rcnn`. These models can not be executed by GE, so it is necessary to pass additioal argument to `evaluate.py`: `--VM`, that means that the model should be inferred by using VM.

Additionaly, the script `evaluate_dyn_models.py` could be used to tune and evaluate static layers from dynamic models (ssd-resnet, yolov3, faster-rcnn). This script was added to compare VM performance vs GE. Below, you can see examples of ssd-resnet evaluation with SD 8 Gen 1 statistics:
```

Also, could you please add a table with performance data for VM?